tools/pate/JSONReport.hs

```diff
@@ -83,9 +114,19 @@ toJSON (SomeProofEvent blks proofSym) = do
                                  , inlineCallError = err
                                  }
       return (JSON.toJSON node)
+    PF.ProofStatus (PF.VerificationFail _counterexample condEqRes)
+      | Nothing <- WI.asConstantPred (PF.condEqPred condEqRes) -> do
```

Comment:
@danmatichuk I'm mostly interested in whether or not this is the right way to detect a differential summary. It seems to work. I was initially checking the `condEqAbortValid` value, but that did not seem to mean what I thought it meant.

Reply:
Actually, now that I think about it, should I actually be matching on a `ProofTriple` node and checking its status (in the way that this is checking status)? That would let me capture the address of the blocks that exhibit the difference

Reply:
Updated to capture the original and patched block addresses